src/api/middleware/validators/company.js

```diff
@@ -55,18 +55,35 @@ const block = useExpressValidators([
         .trim(),
 ]);
 
-const unblock = useExpressValidators([
+const disable = useExpressValidators([
+    param("companyId")
+        .exists().withMessage(ValidationReasons.REQUIRED).bail()
+        .custom(isObjectId).withMessage(ValidationReasons.OBJECT_ID).bail()
+        .custom(companyExists).withMessage(ValidationReasons.COMPANY_NOT_FOUND),
+    body("adminReason")
+        .if((reason, { req }) => req.hasAdminPrivileges).exists().withMessage(ValidationReasons.REQUIRED).bail()
+        .isString().withMessage(ValidationReasons.STRING).bail()
+        .trim(),
+]);
+
+const unhide = useExpressValidators([
```

Comment:
This should match the operation name (enable/disable, not hide/unhide)
```suggestion
const enable = useExpressValidators([
```